Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -122,7 +157,7 @@ public struct Configuration: Equatable {
             let yamlContents = try NSString(contentsOfFile: fullPath,
                 encoding: NSUTF8StringEncoding) as String
             let dict = try YamlParser.parse(yamlContents)
```

Comment:
I sometimes add `print(error)` to `catch` block for knowing the error thrown by `Yam.parse(_:)`…